<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iSoLIM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>Projects</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_prototypes_from_samples"/>
   </widget>
   <widget class="QMenu" name="menuCovariates">
    <property name="title">
     <string>Covariates</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSample_Design">
    <property name="title">
     <string>Sample Design</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSoil_Inference">
    <property name="title">
     <string>Soil Inference</string>
    </property>
    <addaction name="actionFrom_Samples"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <addaction name="actionView_Data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuProjects"/>
   <addaction name="menuCovariates"/>
   <addaction name="menuSample_Design"/>
   <addaction name="menuSoil_Inference"/>
   <addaction name="menuUtilities"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFrom_Samples">
   <property name="text">
    <string>From Samples</string>
   </property>
   <property name="toolTip">
    <string>Soil Inference From Samples</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAdd_prototypes_from_samples">
   <property name="text">
    <string>Add prototypes from samples</string>
   </property>
  </action>
  <action name="actionView_Data">
   <property name="text">
    <string>View Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
